walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,7 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String greeting = "Hello " + name.trim();
```

Comment:
You could try modifying the `makefile` to include week 5's exercises, this way you can run `make pretty` in here as well? https://github.com/shivanighanta/my-portfolio/blob/master/makefile